src/test/java/io/github/bobocodebreskul/server/WebPathScannerTest.java

```diff
@@ -0,0 +1,255 @@
+package io.github.bobocodebreskul.server;
+
+import static io.github.bobocodebreskul.server.enums.RequestMethod.DELETE;
+import static io.github.bobocodebreskul.server.enums.RequestMethod.GET;
+import static io.github.bobocodebreskul.server.enums.RequestMethod.HEAD;
+import static io.github.bobocodebreskul.server.enums.RequestMethod.POST;
+import static io.github.bobocodebreskul.server.enums.RequestMethod.PUT;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchException;
+import static org.mockito.BDDMockito.given;
+
+import io.github.bobocodebreskul.context.exception.AmbiguousHttpAnnotationException;
+import io.github.bobocodebreskul.context.exception.DuplicatePathException;
+import io.github.bobocodebreskul.context.exception.WebPathValidationException;
+import io.github.bobocodebreskul.context.registry.BringContainer;
+import io.github.bobocodebreskul.context.utils.TestUtil;
+import java.lang.reflect.InvocationTargetException;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class WebPathScannerTest {
+
+  @InjectMocks
+  private WebPathScanner webPathScanner;
+  @Mock
+  private BringContainer bringContainer;
+
+  @Test
+  @Order(1)
```

Comment:
That would be great to have human-readable @DisplayName for each test as we did for all tests in the project

Reply:
ok, we can create todo for this